README.md

```diff
@@ -19,11 +19,14 @@ docker run --rm --name osg-frontier-squid \
 Configuring Squid
 -----------------
 
-|                         | Type               | Description                                                                                                               | Example                                   |
-|-------------------------|--------------------|---------------------------------------------------------------------------------------------------------------------------|-------------------------------------------|
-| /etc/squid/customize.sh | Conf file (VOLUME) | One possible way to configure squid is by passing the customize.sh file.                                                  | -v ~/customize.sh:/etc/squid/customize.sh |
-| /etc/squid/squid.conf   | Conf file (VOLUME) | One possible way to configure squid is by passing directly squid.conf file.                                               | -v ~/squid.conf:/etc/squid/squid.conf     |
-| /var/cache/squid        | Cache dir (VOLUME) | This directory contains the cache for squid. If the directory is persistent, the cache will presist after a redeployment. | -v /tmp/squid:/var/cache/squid            |
```

Comment:
This last row is still relevant, i.e. if they want a persistent cache, they need to mount to `/var/cache/squid`